solidity/test/BancorNetwork.js

```diff
@@ -40,79 +40,49 @@ Token network structure:
 */
 
 describe('BancorNetwork', () => {
-    const initPaths = (tokens) => {
-        const bntToken = tokens[0];
-        const erc20Token1 = tokens[1];
-        const erc20Token2 = tokens[2];
-        const erc20Token3 = tokens[3];
-        const anchor1 = tokens[4];
-        const anchor2 = tokens[5];
-        const anchor3 = tokens[6];
-        const anchor4 = tokens[7];
-
-        pathsTokens = {
-            ETH: {
-                BNT: ['', anchor1, bntToken],
-                ERC1: ['', anchor1, bntToken, anchor2, erc20Token1],
-                ERC2: ['', anchor1, bntToken, anchor3, erc20Token2],
-                ERC3: ['', anchor1, bntToken, anchor4, erc20Token3]
-            },
-            BNT: {
-                ETH: [bntToken, anchor1, ''],
-                ERC1: [bntToken, anchor2, erc20Token1],
-                ERC2: [bntToken, anchor3, erc20Token2],
-                ERC3: [bntToken, anchor4, erc20Token3]
-            },
-            ERC1: {
-                ETH: [erc20Token1, anchor2, bntToken, anchor1, ''],
-                BNT: [erc20Token1, anchor2, bntToken],
-                ERC2: [erc20Token1, anchor2, bntToken, anchor3, erc20Token2],
-                ERC3: [erc20Token1, anchor2, bntToken, anchor4, erc20Token3]
-            },
-            ERC2: {
-                ETH: [erc20Token2, anchor3, bntToken, anchor1, ''],
-                BNT: [erc20Token2, anchor3, bntToken],
-                ERC1: [erc20Token2, anchor3, bntToken, anchor2, erc20Token1],
-                ERC3: [erc20Token2, anchor3, bntToken, anchor4, erc20Token3]
-            },
-            ERC3: {
-                ETH: [erc20Token3, anchor4, bntToken, anchor1, ''],
-                BNT: [erc20Token3, anchor4, bntToken],
-                ERC1: [erc20Token3, anchor4, bntToken, anchor2, erc20Token1],
-                ERC2: [erc20Token3, anchor4, bntToken, anchor3, erc20Token2]
-            }
+    const initTokens = async () => {
+        tokens = {
+            ETH: { address: NATIVE_TOKEN_ADDRESS },
```

Comment:
Since we have this, I believe that we no longer need stuff like `reserveToken.address || reserveToken`, `typeof reserveToken === 'string'`, etc.

Reply:
I preferred to keep the "generic" `getBalance` function here as well so that it'd be easier to refactor it everywhere later on (which is one of our future tasks).